app/models/prosecution_case.rb

```diff
@@ -11,14 +11,6 @@ def defendant_ids
     defendants.map(&:id)
   end
 
-  def hearing_summaries
-    body['hearings'].map { |hearing_summary| HearingSummary.new(body: hearing_summary) }
-  end
-
-  def hearing_summary_ids
-    hearing_summaries.map(&:id) if body['hearings'].present?
-  end
-
```

Comment:
Just a question, not a problem, I was just wondering why we're removing the link between `prosecution_cases` and `hearing_summaries`? Is there a better way of modelling them?

Reply:
I removed it because I was going from the Miro board, and didn't find `hearings` listed anywhere in the current design, but maybe I haven't fully understood how they are represented in the UI.
I will check again to be sure.

Reply:
Closing in favour of handling the post linking search later.